lib/sandboxes.ts

```diff
@@ -27,26 +26,18 @@ export async function createSandbox(
   sandbox: Sandbox;
   personalAccessKey: string;
 }> {
```

Comment:
We may need to run it by the sandboxes team, but we could probably just get rid of this file entirely, and the code in the CLI can call the request utils directly

Reply:
I wrote an issue and added it to our board to follow up on removing this file